### PR TITLE
[improve][doc] Add a note of enabled package management on admin-api-packages page

### LINKED ISSUE #14472: upload my jars but it retrun 500.
### What issue do you find in Pulsar docs?

I want use rest api to upload my jars but it retrun 500.

Please perfect the case ,thank you very much.

[the location official documents](https://pulsar.apache.org/docs/en/admin-api-packages/#upload-a-package)

### What is your suggestion?

Please perfect the case ,thank you very much.

### Do you have any references?

Please perfect the case ,thank you very much.

### Would you like to fix this issue?

yes

### Note

- [ ] I have researched my question.

### PR DESCRIPTION
**Pull request by coderzc**
Fixes #14472

### Motivation

Missing a note of enabled package management on admin-api-packages page

### Modifications

Add a note of enabled package management on admin-api-packages page

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `no-need-doc` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-added`
(Docs have been already added)

### CODE REVIEW

File: site2/docs/admin-api-packages.md [hunk 1]
```diff
@@ -60,7 +60,18 @@ source://my-tenant/my-ns/mysql-cdc-source@2.3
...
+You can use the command line tools, REST API, or the Java client to manage your package resources in Pulsar. More specifically, you can use these tools to [upload](#upload-a-package), [download](#download-a-package), and [delete](#delete-a-package) a package, [get the metadata](#get-the-metadata-of-a-package) and [update the metadata](#update-the-metadata-of-a-package) of a package, [get the versions](#list-all-versions-of-a-package) of a package, and [get all packages of a specific type under a namespace](#list-all-packages-of-a-specific-type-under-a-namespace).
+
+To use Package management service, ensure that the package management service has been enabled in your cluster by setting the following properties in broker.conf.
```
Comment (momo-jun) [inline site2/docs/admin-api-packages.md:65]:
```suggestion
To use package management service, ensure that the package management service has been enabled in your cluster by setting the following properties in `broker.conf`.
```

**Comment by momo-jun** [comment]
@coderzc Thanks for contributing to docs and fixing this issue. 
Can you pls request a peer review from a technical perspective?

**Comment by coderzc** [comment]
@codelipenghui can you review the PR?

**Comment by Anonymitaet** [comment]
can you please rename your PR? eg. [improve][doc] xxx
guideline: https://docs.google.com/document/d/1d8Pw6ZbWk-_pCKdOmdvx9rnhPiyuxwq60_TrD68d7BA/edit#heading=h.wu6ygjne8e35

**Comment by coderzc** [comment]
> [improve][doc]

ok

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**